info/v1/container.go

```diff
@@ -266,25 +266,39 @@ type LoadStats struct {
 // CPU usage time statistics.
 type CpuUsage struct {
 	// Total CPU usage.
-	// Units: nanoseconds
+	// Unit: nanoseconds.
 	Total uint64 `json:"total"`
 
 	// Per CPU/core usage of the container.
 	// Unit: nanoseconds.
 	PerCpu []uint64 `json:"per_cpu_usage,omitempty"`
 
 	// Time spent in user space.
-	// Unit: nanoseconds
+	// Unit: nanoseconds.
 	User uint64 `json:"user"`
 
 	// Time spent in kernel space.
-	// Unit: nanoseconds
+	// Unit: nanoseconds.
 	System uint64 `json:"system"`
 }
 
+// Cpu Completely Fair Scheduler statistics.
+type CpuCFS struct {
+	// Total number of elapsed enforcement intervals.
+	Periods uint64 `json:"periods"`
```

Comment:
If `cpu.cfs_quota` is set to a value greater -1, the CPU usage of that cgroup will be limited to that amount of nanoseconds per period. The period length is defined by `cpu.cfs_period_us`. Both of these values get exported already under the `spec` namespace. The periods here describe how many periods any process in the cgroup requested some cpu cycles. As far as I understand, a completely idle cgroup will not cause this counter to go up.